mitosheet/mitosheet/public/v3/sheet_functions/number_functions.py

```diff
@@ -119,6 +121,7 @@ def CORR(s1: NumberRestrictedInputType, s2: NumberRestrictedInputType) -> Number
         "function": "CORR",
         "description": "Computes the correlation between two series, excluding missing values.",
         "search_terms": ["corr", "correlation", "r^2"],
+        "category": "MATH",
```

Comment:
Quick callout that we should make sure to update these docs accordingly: https://docs.trymito.io/how-to/interacting-with-your-data/bring-your-own-spreadsheet-functions#displaying-syntax-hints-inside-of-mito

The dev could either set a category so their custom functions show up in the correct bucket, and/or we could create a `custom functions` category so the dev can have their functions all appear in one section. 

If they write a custom function and don't add a `category` then it should appear in the custom functions button.